### PR TITLE
Shipment

### PR DESCRIPTION
**Pull request by pichar376**
shipment page

### CODE REVIEW

File: src/components/MenuDelivery/MenuDelivery.jsx [hunk 1]
```diff
@@ -48,15 +53,42 @@ const StyledMenu = styled((props) => (
...
+
   const handleClose = () => {
     setAnchorEl(null);
   };
+  const { update } = DeliveryStorageService;
+
+  const deleteRow = (id) => {
+    const rowToDelete = deliveryHistory.filter((el) => el.orderId === id);
+    const filterRows = deliveryHistory.filter(
+      (el) => el.orderId !== rowToDelete[0].orderId
```
Comment (r3ivaj) [inline src/components/MenuDelivery/MenuDelivery.jsx:82]: With this change the filter above is not needed
```suggestion
      (el) => el.orderId !== id
```

File: src/pages/Shipment/Shipment.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { useParams } from "react-router-dom";
+
+const Shipment = () => {
+  let { id } = useParams();
+
+  console.log(id);
```
Comment (r3ivaj) [inline src/pages/Shipment/Shipment.jsx:8]: please remove this console log

File: src/components/MenuDelivery/MenuDelivery.jsx (hunk 1 above)
Comment (pichar376) [reply src/components/MenuDelivery/MenuDelivery.jsx:82]: Oh very true.

File: src/pages/Shipment/Shipment.jsx (hunk 2 above)
Comment (pichar376) [reply src/pages/Shipment/Shipment.jsx:8]: Ok.

**Pull request merged**